<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="open_LED">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开LED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_LED">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关LED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_RELAY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开门禁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_RELAY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关门禁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_FAN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开风扇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_FAN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关风扇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_BUZZER">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开蜂鸣器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_BUZZER">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关蜂鸣器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_camera">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="face_train">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>录入人脸</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
